bridge_integration_test.go

```diff
@@ -978,10 +975,10 @@ func setupInjectedToolTest(t *testing.T, fixture []byte, streaming bool, configu
 	recorderClient := &mockRecorderClient{}
 
 	// Setup MCP tools.
-	tools := setupMCPServerProxiesForTest(t)
+	tools := setupMCPServerProxiesForTest(t, testTracer)
```

Comment:
Nit: it's confusing to name this tools, since it's not actually a list of tools.